src/manifest/dbtProjectContainer.ts

```diff
@@ -134,12 +134,14 @@ export class DBTProjectContainer implements Disposable {
     );
     if (answer === PromptAnswer.YES) {
       commands.executeCommand("dbtPowerUser.openSetupWalkthrough");
+    } else if (answer === PromptAnswer.IGNORE) {
+      this.setToGlobalState("showSetupWalkthrough", false);
     }
```

Comment:
```suggestion
    if (answer === PromptAnswer.YES) {
      commands.executeCommand("dbtPowerUser.openSetupWalkthrough");
    } 
    this.setToGlobalState("showSetupWalkthrough", false);
```

Reply:
doing this will set the flag even if user clicks on the x mark to close the prompt instead of clicking on the button ignore. X mark should mean ignore the prompt and not take action so doing this.